### DISCUSSION TITLE
Using FLARE for the system containing three species?

### DISCUSSION THREAD

**Discussion by hewuanhewuan** [mir-group/flare#404] [Q&A]
Hello,

1. I want to train a reaxff force field for Pd/H/O system. After my several attempts, the results were all terrible. I also trained several independent systems (H2, O2, H2O2, H2O, Pd bulk, Pd surface and Pd/H/O) according to the paper. Are these systems rational to train Pd/H/O reaxff force field? I wonder if FLARE has the ability to train a system containing 3 species despite the successful application on Pt/H in the paper.
2. When I used offline training to pool together several systems, I set the train_hyps to be infinitely large so that the hyperparameters are not optimized. Does it mean all the frames will use the hyperparameters I initially set in the yaml file? Are these fixed hyperparameters suitable for all the 7 independent systems I mentioned before？Or I also have to train hyperparameters during the offline training？

I really need someone to teach me how to train this force field, and I almost broke down in this process.

Thank you so much.

Best,
wuan

**Comment by YuuuXie** [comment]
Hi,

1. yes flare can handle three species, I don’t think it is a problem, just requires more data. I think the systems you choose look ok. 
2. Yes your understanding is correct. If you don’t train the hyper parameters, then they will be fixed to your initial value. You can set them to e.g. your estimated errors of energy, force, stress, corresponding to the energy/force/stress noise hyperparameter
